tdrs-backend/tdpservice/users/models.py

```diff
@@ -101,6 +101,9 @@ class User(AbstractUser):
             "Users in an approved state are allowed access."
         ),
     )
+
+    access_requested_date = models.DateTimeField(null=True)
```

Comment:
@elipe17 this field is initialized as empty for any pre-existing users, which means that the email will not include those with empty access request date. Do you propose that we set these manually for any accounts in the access_request state when this code is pushed to prod? 
![1336p2](https://user-images.githubusercontent.com/63075587/234704752-2d1c0a0e-3e87-478f-a196-1212d79baf1f.PNG)

Reply:
Good question. As it stands, the email only includes new access requests from the last 24 hours. So there are three options. One, If we wanted a default value for the datetime field and you wanted to receive an email with those users we would have to pick a default that fell into the 24hr period that occurred when pushed to prod. Two, we could set some default value and have the email collect all users in the access request state regardless of how long ago they requested access. Three, we could go with your manual suggestion and impute a datetime once the code has been pushed to prod. Let me know your thoughts on the options.

Reply:
@elipe17 i vote for option 2. i dont mind the manual approach, but i think we still want to keep track of all requests, regardless of when they made the request.

Reply:
Sounds good, I will make that change.